azure-pipelines.yml

```diff
@@ -116,6 +116,87 @@ stages:
           flake8 --select F401 great_expectations tests
         name: UnusedImportChecker
 
+  - stage: docker_tests
+    dependsOn: []
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - job: build_push_test_
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Python37:
+              minor_version: '7'
+            Python38:
+              minor_version: '8'
+            Python39:
+              minor_version: '9'
+        services:
+          postgres: postgres
+        variables:
+          DOCKER_TAG: none
+        steps:
+          - bash: |
```

Comment:
Is there value in moving these commands to separate scripts? I like being able to read it all in one place but it does get a bit verbose